hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1221,7 +1224,6 @@ def minimize_floats(self, chooser):
             and not is_simple(node.value),
         )
 
-        i = self.nodes.index(node)
         # the Float shrinker was only built to handle positive floats. We'll
```

Comment:
I realized that tracking an index on `IRNode` is safer than `self.nodes.index(node)`, because the latter can get tripped up by equality and return a different index.